repo/fsrepo/migrations/migrations.go

```diff
@@ -0,0 +1,261 @@
+package mfsr
+
+import (
+	"bufio"
+	"bytes"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"net/http"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"runtime"
+	"strconv"
+	"strings"
+)
+
+var DistPath = "https://ipfs.io/ipfs/QmUnvqDuRyfe7HJuiMMHv77AMUFnjGyAU28LFPeTYwGmFF"
```

Comment:
doesn't this need a migration usable by `0.4.3(-*)?` ? i dont think this has the last migration needed